MapleServer2/PacketHandlers/Game/UserSyncHandler.cs

```diff
@@ -68,12 +68,26 @@ public override void Handle(GameSession session, PacketReader packet)
 
         private static bool IsOutOfBounds(CoordF coord, CoordS[] boundingBox)
         {
-            CoordS higherBoundingBox = boundingBox[0].Z > boundingBox[1].Z ? boundingBox[0] : boundingBox[1];
-            CoordS lowerBoundingBox = boundingBox[0].Z > boundingBox[1].Z ? boundingBox[1] : boundingBox[0];
-            if (coord.Z > higherBoundingBox.Z || coord.Z < lowerBoundingBox.Z)
+            short HigherBoundZ = boundingBox[0].Z > boundingBox[1].Z ? boundingBox[0].Z : boundingBox[1].Z;
```

Comment:
Why not use math functions here

Reply:
Not sure what you mean? You can't do > or < with a Vector3. 
I could only do only two checks instead of six, but I didn't want to trust XML consistency.

Reply:
These variables should be lowerCamelCase also

Reply:
And yeah we can't just > or < on the entire coord object because a Z might be higher in box 0 while having a lower Y, etc.

Reply:
But I'm not saying use it on the entire coord object, he's only using it on the Z property here, meaning you should be able to use `Math.Max`:

https://docs.microsoft.com/en-us/dotnet/api/system.math.max?view=net-5.0